tiatoolbox/models/architecture/utils.py

```diff
@@ -21,37 +21,90 @@
 """Defines utlity layers and operators for models in tiatoolbox."""
 
 
+from typing import Union
+
 import numpy as np
 import torch
 import torch.nn as nn
 
 
-def crop_op(x, cropping, data_format="NCHW"):
-    """Center crop image with substracted amount.
+def center_crop(
```

Comment:
use British or US English not a mix of both.
I think we are using British spellings so use centre then